server/odc-service/src/main/java/com/oceanbase/odc/service/sqlcheck/rule/MySQLAffectedRowsExceedLimit.java

```diff
@@ -76,17 +81,42 @@ public SqlCheckRuleType getType() {
     @Override
     public List<CheckViolation> check(@NonNull Statement statement, @NonNull SqlCheckContext context) {
 
+        long affectedRows = getAffectedRows(statement);
+        if (affectedRows >= 0) {
+            if (affectedRows > maxSqlAffectedRows) {
+                return Collections.singletonList(SqlCheckUtil
+                        .buildViolation(statement.getText(), statement, getType(),
+                                new Object[] {maxSqlAffectedRows, affectedRows}));
+            } else {
+                return Collections.emptyList();
+            }
+        }
+        return Collections.emptyList();
+    }
+
+    /**
+     * Get supported database types
+     */
+    @Override
+    public List<DialectType> getSupportsDialectTypes() {
+        return Arrays.asList(DialectType.MYSQL, DialectType.OB_MYSQL);
+    }
+
+    protected long getAffectedRows(@NonNull Statement statement) {
+
```

Comment:
we may remove these meaningless breaking lines, same as below

Reply:
got it